native/python/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I recall @t1m0thyj considering if we should use `make` for the build if others feel the same, perhaps we should be consistent on build .  I'll defer to group consensus